test/test_client_async.py

```diff
@@ -7,14 +7,14 @@
     # vendored backport module
     import kafka.vendor.selectors34 as selectors
 
+import pytest
```

Comment:
this should not be moved per pep8, `pytest` should be in 3p library imports separate from stdlib imports

Reply:
Noted. I'll change